include/pacemaker.h

```diff
@@ -47,7 +47,8 @@ typedef enum {
     show_tickets       = 1 << 14,
     show_bans          = 1 << 15,
     show_failures      = 1 << 16,
-    show_timings       = 1 << 17
+    show_timings       = 1 << 17,
+    show_rscs_by_node  = 1 << 18
```

Comment:
FYI the last item can have a comma, that avoids extra diff lines for later additions

Reply:
Fixed.  That's good to know.